angr/analyses/decompiler/decompilation_options.py

```diff
@@ -198,6 +198,16 @@ def __repr__(self):
         default_value=True,
         clears_cache=True,
     ),
+    O(
+        "Simple Statement Compare",
```

Comment:
Let's change all the references of this to be something like "c-style ifs", since this feature introduces a new way to display syntax in a C form, not a "simple statement compare"